geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+complexes: a comparative study of a weak C–H⋅⋅⋅N hydrogen bond and an S⋅⋅⋅N bond. Phys Chem Chem Phys
+13(31):14235-14242
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+S...N   3.3233(2)
+NH3 SCO 4.3607(2)
+α       25.23(4)
+
+# α: Angular oscillation of the ammonia subunit. 
+
+NAME = CH3O 
+# Point Group: C3v
+# Shape: Seesaw
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Liu J, Chen MW, Melnik D, Miller TA, Endo Y, Hirota E (2009) The spectroscopic characterization of the
+methoxy radical. II. Rotationally resolved A2A1-X2E electronic and X2E microwave spectra of the perdeuteromethoxy radical CD3O. J Chem Phys 130(7):074303/1-074303/11
+https://doi.org/10.1063/1.2955739
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C O   1.36039(8)
+C H   1.10697(9)
+H C H 107.76(1)
+
+NAME = CH4Zn 2010
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Flory MA, Apponi AJ, Zack LN, Ziurys LM (2010) Activation of methane by zinc: Gas-phase synthesis,
+structure, and bonding of HZnCH3. J Amer Chem Soc 132(48):17186-17192
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+H  Zn  1.5209(1)
+Zn C   1.9281(2)
+C  H   1.140(9)
+Zn C H 110.2(3)
+H  C H 108.7(3)
+
+NAME = C2AgCl  2015
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Zaleski DP, Tew DP, Walker NR, Legon AC (2015) Chemistry in laser-induced plasmas: formation of M-C≡CCl (M = Ag or Cu) and their characterization by rotational spectroscopy. J Phys Chem A 119(12):2919-2925
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Ag   C  2.015(14)
+C  ≡ C  1.2219
+C    Cl 1.635(6)
+Ag...Cl 4.8722(2)
+
+# Bond length C ≡ C 1.2219 assumed at the CCSD(T)/aug-cc-pV5Z value
+
+# Bond lengths: rs (angstroms)
+
+Ag...Cl 4.8729(12)
+
+NAME = C2ClCu  2015
+# Point Group: Coov 
+# Shape: Linear
+# Bond lengths: r0 (angstroms)
+# Source paper: Zaleski DP, Tew DP, Walker NR, Legon AC (2015) Chemistry in laser-induced plasmas: formation of M-C≡CCl (M = Ag or Cu) and their characterization by rotational spectroscopy. J Phys Chem A 119(12):2919-2925
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Cu   C  1.812(16)
+C  ≡ C  1.2233
+C    Cl 1.639(6)
+Cu...Cl 4.6736(2)
+
+# Bond length C ≡ C 1.2233 assumed at the CCSD(T)/aug-cc-pV5Z value
+
+# Bond lengths: rs (angstroms)
+
+CU...Cl 4.6743(10)
+
+NAME = C2F3IO
+# Point Group: C3v
+# Shape: Tetrahedral --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Stephens SL, Walker NR, Legon AC (2011) Rotational spectra and properties of complexes B⋅⋅⋅ICF3 (B = Kr or
+CO) and a comparison of the efficacy of ICl and ICF3 as iodine donors in halogen bond formation. J Chem Phys
+135(22):224309/1-224309/8
+doi:10.1063/1.3664314
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Rcm   4.9644(12)
+I...C 3.4281(12)
+ϕ     4.0(5)
+φ     10(3)
+
```

Comment:
Instead of Rcm, please put CO F3I. These angles also need to be explained somehow.

Reply:
I'm kind of confused with the justification for ϕ and φ. 
![image](https://github.com/HPQC-LABS/goDatabase/assets/123907626/55e6e501-cf96-472d-9cb6-052f7edaf23c)
Are they reusing the notation Rcm in this case to denote the line connecting the centers of mass?